aws/eks/fargate/spot_and_fargate/fargate_eks/main.tf

```diff
@@ -0,0 +1,249 @@
+# MIT License
```

Comment:
Do we need a copyright notice on this?

Reply:
I don't know. Added it because I copied the code from an existing example. Maybe just mentioning on the README.md is sufficient